packages/@react-aria/test-utils/src/menu.ts

```diff
@@ -241,6 +241,17 @@ export class MenuTester {
           throw new Error('Expected menu element to not be in the document after selecting an option');
         }
       }
+      // else {
+      //   // TODO: this is a bit unfortunate, but from a submenu point of view, we can't perform a check that waits for
+      //   // focus to return to the original trigger since we don't have that information in the submenu trigger tester. Even if we did
+      //   // it is a bit unpredicatable where focus might end up landing and waiting for the option/menu to disappear isn't sufficient if there are
+      //   // more transitions in play. For now advance times by a certain amount of time and rely on the user to advance them even more if need be
+      //   if (this._advanceTimer == null) {
+      //     throw new Error('No advanceTimers provided for long press.');
+      //   } else {
+      //     await act(async () => await this._advanceTimer(1000));
+      //   }
+      // }
```

Comment:
Did some poking around trying to get rid of the last `jest.runAllTimers()` here but no luck. The above is ok, but can extend test duration by quite a bit, also seemed to have some problems with hanging that I didn't quite figure out